stubs/enterReplies.js

```diff
@@ -1 +1,27 @@
-module.exports = ['Hi', 'Target Acquired', 'Firing', 'Hello friend.', 'Gotcha', 'I see you']
\ No newline at end of file
+introduction = 'Please join hackbot in welcoming @user to the Rogue Hack Lab!!! :tada:';
+
+icebreakers = [
+  'What\'s your favorite 1990\'s cartoon?',
+  'What did you have for breakfast?',
+  'What\'s one skill you would like to develop?',
+  'What\'s your favorite letter of the alphabet and why?',
+  'How old were you when you first used a computer?',
+  'What\'s your favorite spot for a beer?',
+  'How did you hear about the hack lab?',
+  'When was the last time you changed your hairstyle?',
+  'Have you committed today?',
+  'You\'re trapped on a deserted island, what TV show are you bringing to keep you company?',
+  'What\'s your favorite flavor of sparkling water?',
+  'Tell us your favorite \'nerdy\' joke',
+  'How old were you when you stopped believing in Santa Claus?',
```

Comment:
Wait what?

Reply:
My bad, I should have prefaced this with LIFE SPOILER ALERT.

Reply:
Sparkling water, a nerd, and Santa Clause walk into a bar...